packages/vx-shape/src/shapes/SplitLinePath/getLineSegments.ts

```diff
@@ -0,0 +1,69 @@
+import memoize from 'lodash/memoize';
+
+const MEASUREMENT_ELEMENT_ID = '__vx_splitpath_svg_path_measurement_id';
+
+export interface GetLineSegmentsConfig<Datum> {
+  /** Full path `d` attribute to be broken up. */
+  path: string;
+  segments: Datum[][];
+  sampleRate?: number;
+}
+
+type LineSegments = { x: number; y: number }[][];
+
+export function getLineSegments<Datum>({
+  path,
+  segments,
+  sampleRate = 0.25,
+}: GetLineSegmentsConfig<Datum>): LineSegments {
+  try {
+    let pathElement = document.getElementById(MEASUREMENT_ELEMENT_ID) as SVGPathElement | null;
+
+    // create a single path element if not done already
+    if (!pathElement) {
+      const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+      svg.style.width = '0';
+      svg.style.height = '0';
+      svg.style.position = 'absolute';
+      svg.style.top = '-100%';
+      svg.style.left = '-100%';
+      pathElement = document.createElementNS('http://www.w3.org/2000/svg', 'path');
```

Comment:
can extract the namespace url to constant

Reply:
Perhaps add

```ts
// not visible
svg.style.opacity = 0;
// won't capture mouse events
svg.style.pointerEvents = 'none';
``` 

for extra precaution

Reply:
^was wondering about this. we don't do this in `@vx/text` and I forget if there was a reason. will search for old PRs